ts/ui/menu/Menu.ts

```diff
@@ -473,9 +473,9 @@ export class Menu {
         this.submenu('Accessibility', 'Accessibility', [
           this.checkbox('Activate', 'Activate', 'explorer'),
           this.submenu('Speech', 'Speech', [
-            this.checkbox('Speech', 'Speech Output', 'speech'),
-            this.checkbox('Subtitles', 'Speech Subtitles', 'subtitles'),
-            this.checkbox('Braille', 'Braille Output', 'braille'),
+            this.checkbox('Speech', 'Speech Output', 'speechExp'),
+            this.checkbox('Subtitles', 'Speech Subtities', 'subtitles'),
```

Comment:
You've put back the typo that was fixed in an earlier PR.

Reply:
Well spotted! How did that happen?

Reply:
Probably in the merge of develop into this branch.  Sometimes It may think this file is more current and so took its changes over the ones from develop.  Don't know for sure.